Readme.md

```diff
@@ -115,6 +115,10 @@ $ npm install
 $ node examples/content-negotiation
```

Comment:
Am I missing it in here or are we not linking all the docs? Are we consolidating them in the admin repo so we don't need to link them all individually?

Reply:
this folder is full of examples with multiple implementations (and with different modules). I think it would be great to move all that in the https://github.com/expressjs/examples (or somewhere else)

Reply:
sorry, that was not meant as a comment on the line specifically but the entire change in this file. I don't see all the docs linked, just a couple. Are we going to simplify them in the new repo? Or do we need to add more links to this readme change?